### PR TITLE
#168: Add possibility to set 2 ISO values: preview and capture

### PR DESCRIPTION
**Pull request by anxuae**

### CODE REVIEW

**Comment by anxuae** [comment]
Shall we keep the `gphoto_omx.py` implementation? I try to update it when something change in the cameras classes, but it is hard to say if it is correct at this class has never been finished.

**Review by werdeil — COMMENTED** [review]
I think we shall be explicit in the code names: iso_preview and iso_capture
And reserve iso for the config / tuple

**Comment by werdeil** [comment]
> Shall we keep the `gphoto_omx.py` implementation? I try to update it when something change in the cameras classes, but it is hard to say if it is correct at this class has never been finished.

I think we can drop it. It was only an exploration

File: pibooth/camera/opencv.py [hunk 1]
```diff
@@ -217,6 +219,10 @@ def capture(self, effect=None):
         self._cam.set(cv2.CAP_PROP_FRAME_WIDTH, self.preview_resolution[0])
         self._cam.set(cv2.CAP_PROP_FRAME_HEIGHT, self.preview_resolution[1])
 
+        if self.iso_capture != self.iso_preview:
+            self._cam.set(cv2.CAP_PROP_ISO_SPEED, self.iso_preview)
+            print(self.iso_preview)
```
Comment (werdeil) [inline pibooth/camera/opencv.py:224]: I think you left a debug print here

File: pibooth/camera/base.py [hunk 2]
```diff
@@ -9,14 +9,18 @@
...
         self._border = 50
         self._window = None
         self._overlay = None
         self._captures = []
         self.resolution = resolution
         self.delete_internal_memory = delete_internal_memory
+        if not isinstance(iso, (tuple, list)):
+            iso = (iso, iso)
+        self.iso_preview = iso[0]
+        self.iso_capture = iso[1]
```
Comment (werdeil) [inline pibooth/camera/base.py:23]: Up to you but we can change it to:
```suggestion
        self.iso_preview, self.iso_capture = iso
```

**Review by werdeil — CHANGES_REQUESTED** [review]
two small comments

File: pibooth/camera/opencv.py (hunk 1 above)
Comment (anxuae) [reply pibooth/camera/opencv.py]: Corrected

**Pull request merged**